### PR TITLE
COO-225: fix: Add support for ConsolePlugin v1

### PR DESCRIPTION
**Pull request by jgbernalp**
This PR adds support for ConsolePlugin CR v1 for OCP 4.17. For older versions the v1alpha1 support was kept

### CODE REVIEW

File: pkg/controllers/uiplugin/components.go [hunk 1]
```diff
@@ -45,14 +48,32 @@ var (
 	korrel8rConfigYAMLTmplFile embed.FS
 )
 
-func pluginComponentReconcilers(plugin *uiv1alpha1.UIPlugin, pluginInfo UIPluginInfo) []reconciler.Reconciler {
+func isClusterVersionAheadOrEqual(clusterVersion, nextClusterVersion string) bool {
```
Comment (lihongyan1) [inline pkg/controllers/uiplugin/components.go:51]: a nitpicking, feels the variable names of isClusterVersionAheadOrEqual is unreasonable and confusing, maybe better change as `func isClusterVersionAheadOrEqual(currentClusterVersion, clusterVersion string) bool {`

**Comment by lihongyan1** [comment]
Verified the bug COO-225 with this PR, the issue is gone on OCP4.17

File: pkg/controllers/uiplugin/components.go (hunk 1 above)
Comment (jan--f) [reply pkg/controllers/uiplugin/components.go:51]: Yeah I agree, the naming is awkward currently. I see how future devs might think the clusterVersion in the name is related to the variable name.

Comment (jgbernalp) [reply pkg/controllers/uiplugin/components.go]: updated to `func isVersionAheadOrEqual(currentVersion, version string) bool {`

**Comment by jan--f** [comment]
/lgtm
Thanks!

**Pull request merged**